### DISCUSSION TITLE
Welcome to cl-liballegro Discussions!

### DISCUSSION THREAD

**Discussion by resttime** [resttime/cl-liballegro#26] [General]
Apparently Github discussions is a thing now, gonna enable this for the repo to try it out

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by knerking** [comment]
Hi. Thanks for cl-liballegro.
This time, when I try to install it with quicklisp,
it aborts because mm_malloc.h is missing in the gcc it uses for compiling things.
The gcc is the latest version in msys2.
Do you know what I can do about it?

Regards Knerking

**Reply by lockie** [reply]
I had no trouble with liballegro under MSYS. Which MSYS distribution are you using? And how the error message looks exactly?

**Comment by knerking** [comment]
Hi.
Sorry. It was a problem with my windows PATH. The wrong version of gcc was
executed.
Now it is installed. And thank you again for your work with cl-liballegro.

Regards Mikael

Den mån 26 juni 2023 kl 10:42 skrev Andrew Kravchuk <
***@***.***>:
